src/excmds.ts

```diff
@@ -5096,6 +5096,15 @@ export async function extoptions(...optionNameArgs: string[]) {
     return winopen("-popup", selectedExtension.optionsUrl)
 }
 
+//#content_helper
+import { Readability } from "@mozilla/readability"
+
+//#content
+export async function reader2() {
+    const article = new Readability(document).parse()
+    document.body.innerHTML = article.content
```

Comment:
Probably easiest to make a data uri - https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs - out of `article.content` and then use `open(` to open it. or maybe just do `document.location.href = [data URI]`. That way we'll get a useful back button.